spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -4,9 +4,9 @@
   let(:user) { build(:user) }
   let(:profile) { build(:profile, user: user) }
   let(:current_address_matches_id) { true }
-  let(:created_at) { described_class::USPS_SERVER_TIMEZONE.parse('2022-07-14T00:00:00Z') }
+  let(:created_at) { described_class::USPS_SERVER_TIMEZONE.parse('2023-06-14T00:00:00Z') }
   let(:enrollment_established_at) do
-    described_class::USPS_SERVER_TIMEZONE.parse('2022-08-14T00:00:00Z')
+    described_class::USPS_SERVER_TIMEZONE.parse('2023-07-14T00:00:00Z')
```

Comment:
The purpose of these changes is not obvious to me. Was this ultimately just to get us away from an edge-of-DST-change date?

Reply:
That's exactly the case. As far as I can tell, September 11- September 12th is DST change for UTC -300. So changing by a year and a month.